### ISSUE TITLE
[YouTube] Download multiple playlists but doesn't start from the top 

### ISSUE THREAD

**Issue by Wilann** [yt-dlp/yt-dlp#11656]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting that yt-dlp is broken on a **supported** site
- [X] I've verified that I have **updated yt-dlp to nightly or master** ([update instructions](https://github.com/yt-dlp/yt-dlp#update-channels))
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/yt-dlp/yt-dlp/wiki/FAQ#video-url-contains-an-ampersand--and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

Canada

### Provide a description that is worded well enough to be understood

I have been using this script to download all playlists from this channel. Before it always started at the top - in the image below, that would be "Syed Modi India International 2024" and go right to left (playlist with latest added video --> playlist with earliest added video), as expected. 

Now, it's changed to start at "Orleans Masters Badminton presented by VICTOR 2024", which is 5 rows down (or 30 playlists later). 

Not sure what the issue is or what's changed, since I haven't modified this code for months now. 

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [ ] If using API, add `'verbose': True` to `YoutubeDL` params instead
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
[debug] Command-line config: ['-P', 'temp:/tmp/yt-dlp', '-P', 'home:/home/USERNAME/Giratina/BWF TV (youtube-dl)', '-o', '%(playlist_title)s/%(upload_date)s - %(title)s', '--config-location', '/home/USERNAME/Giratina/Project/Live/config_video.txt', '--download-archive', '/home/USERNAME/Giratina/BWF TV (youtube-dl)/archive.txt', 'https://www.youtube.com/@bwftv/playlists', '--verbose']
[debug] | Config "/home/USERNAME/Giratina/Project/Live/config_video.txt": ['--match-filter', '!is_live', '--format', 'bestvideo[ext=mp4]+bestaudio[ext=m4a]/bestvideo[vcodec!=vp9]+bestaudio[ext=m4a]/best', '--merge-output-format', 'mp4', '--add-metadata', '--embed-thumbnail', '--write-thumbnail', '--no-write-playlist-metafiles', '--retries', 'inf', '--fragment-retries', 'inf']
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version stable@2024.11.18 from yt-dlp/yt-dlp [7ea278792] (pip)
[debug] Python 3.9.7 (CPython x86_64 64bit) - Linux-6.0.12-76060012-generic-x86_64-with-glibc2.31 (OpenSSL 1.1.1l  24 Aug 2021, glibc 2.31)
[debug] exe versions: ffmpeg 4.2.7, ffprobe 4.2.7
[debug] Optional libraries: Cryptodome-3.13.0, brotli-None, certifi-2021.10.08, mutagen-1.45.1, requests-2.32.3, secretstorage-3.3.1, sqlite3-3.36.0, urllib3-2.1.0, websockets-12.0
[debug] Proxy map: {}
[debug] Request Handlers: urllib, requests
[debug] Loaded 1837 extractors
[debug] Loading archive file '/home/USERNAME/Giratina/BWF TV (youtube-dl)/archive.txt'
[youtube:tab] Extracting URL: https://www.youtube.com/@bwftv/playlists
[youtube:tab] @bwftv/playlists: Downloading webpage
[debug] [youtube:tab] Selected tab: 'playlists' (playlists), Requested tab: 'playlists'
[download] Downloading playlist: BWF TV - Playlists
[youtube:tab] UChh-akEbUM8_6ghGVnJd6cQ page 1: Downloading API JSON
[youtube:tab] UChh-akEbUM8_6ghGVnJd6cQ page 2: Downloading API JSON
[youtube:tab] UChh-akEbUM8_6ghGVnJd6cQ page 3: Downloading API JSON
[youtube:tab] UChh-akEbUM8_6ghGVnJd6cQ page 4: Downloading API JSON
[youtube:tab] UChh-akEbUM8_6ghGVnJd6cQ page 5: Downloading API JSON
[youtube:tab] UChh-akEbUM8_6ghGVnJd6cQ page 6: Downloading API JSON
[youtube:tab] UChh-akEbUM8_6ghGVnJd6cQ page 7: Downloading API JSON
[youtube:tab] UChh-akEbUM8_6ghGVnJd6cQ page 8: Downloading API JSON
[youtube:tab] UChh-akEbUM8_6ghGVnJd6cQ page 9: Downloading API JSON
[youtube:tab] UChh-akEbUM8_6ghGVnJd6cQ page 10: Downloading API JSON
[youtube:tab] UChh-akEbUM8_6ghGVnJd6cQ page 11: Downloading API JSON
[youtube:tab] UChh-akEbUM8_6ghGVnJd6cQ page 12: Downloading API JSON
[youtube:tab] UChh-akEbUM8_6ghGVnJd6cQ page 13: Downloading API JSON
[youtube:tab] Playlist BWF TV - Playlists: Downloading 374 items of 374
[download] Downloading item 1 of 374
[youtube:tab] Extracting URL: https://www.youtube.com/playlist?list=PLA7ZcagI0frDZAftqqREGbPJ31It4PLh8
[youtube:tab] PLA7ZcagI0frDZAftqqREGbPJ31It4PLh8: Downloading webpage
[youtube:tab] PLA7ZcagI0frDZAftqqREGbPJ31It4PLh8: Redownloading playlist API JSON with unavailable videos
[download] Downloading playlist: Orleans Masters Badminton presented by VICTOR 2024
[youtube:tab] PLA7ZcagI0frDZAftqqREGbPJ31It4PLh8 page 1: Downloading API JSON
WARNING: [youtube:tab] Incomplete data received. Retrying (1/3)...
[youtube:tab] PLA7ZcagI0frDZAftqqREGbPJ31It4PLh8 page 1: Downloading API JSON
WARNING: [youtube:tab] Incomplete data received. Retrying (2/3)...
[youtube:tab] PLA7ZcagI0frDZAftqqREGbPJ31It4PLh8 page 1: Downloading API JSON
WARNING: [youtube:tab] Incomplete data received. Retrying (3/3)...
[youtube:tab] PLA7ZcagI0frDZAftqqREGbPJ31It4PLh8 page 1: Downloading API JSON
WARNING: [youtube:tab] Incomplete data received. Giving up after 3 retries
[youtube:tab] Playlist Orleans Masters Badminton presented by VICTOR 2024: Downloading 29 items of 29
```

**Comment by bashonly** [comment]
Most likely a side effect of fe70f20aedf528fdee332131bc9b6710e54e6f10 which fixed #11524. Since this was a change on Youtube's end, I'm not sure if it's possible to restore the original behavior

**Comment by bashonly** [comment]
Fixed by fe70f20aedf528fdee332131bc9b6710e54e6f10
